client/my-sites/subscribers/subscriber-list/subscriber-row.tsx

```diff
@@ -0,0 +1,42 @@
+import { Gridicon } from '@automattic/components';
+import moment from 'moment';
+import FormCheckbox from 'calypso/components/forms/form-checkbox';
+import { Subscriber } from '../types';
+import { SubscriberProfile } from './subscriber-profile';
+
+type SubscriberRowProps = Subscriber & {
+	locale: string;
+};
+
+export const SubscriberRow = ( {
+	display_name,
+	email_address,
+	subscriptions,
+	openRate,
+	date_subscribed,
+	avatar,
+	locale,
+}: SubscriberRowProps ) => {
+	return (
+		<li className="subscriber-row row" role="row">
+			<div className="subscriber-list__checkbox-column hidden" role="cell">
+				<FormCheckbox />
+			</div>
+			<span className="subscriber-list__profile-column" role="cell">
+				<SubscriberProfile avatar={ avatar } displayName={ display_name } email={ email_address } />
+			</span>
+			<span className="subscriber-list__subscription-type-column hidden" role="cell">
+				{ subscriptions && subscriptions.map( ( subscription ) => <div>{ subscription }</div> ) }
+			</span>
+			<span className="subscriber-list__rate-column hidden" role="cell">
+				{ openRate }
+			</span>
+			<span className="subscriber-list__since-column" role="cell">
+				{ moment( date_subscribed ).locale( locale ).format( 'LL' ) }
+			</span>
```

Comment:
I think we could consider using the `<TimeSince />` component here. This will allow us to get rid of passing the locale down the component tree and make the code consistent with what we already have in the _Subscriptions Management_ portal (e.g. in `client/landing/subscriptions/components/site-subscriptions-manager/site-row.tsx`)